### PR TITLE
Video link for better understanding

### PR DESCRIPTION
**Pull request by Gamingspider**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [ ] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by Gamingspider** [comment]
YouTube link for better understanding

File: guide/english/python/class/index.md [hunk 1]
```diff
@@ -112,4 +112,4 @@ class Friend:
 fr1 = Friend("Nilesh","Bharti","Bihar")
 fr1.show()
 ```
-
+href="https://www.youtube.com/watch?v=ZDa-Z5JzLYM&feature=youtu.be " target="_blank"><img src="http://img.youtube.com/vi/v=ZDa-Z5JzLYM/0.jpg" alt="CLASS" width="240" height="180" border="10" /></a>
```
Comment (rupali317) [inline guide/english/python/class/index.md]: This should begin with **<a** href="https.... right?

**Review by rupali317 — CHANGES_REQUESTED** [review]

File: guide/english/python/class/index.md (hunk 1 above)
Comment (RandellDawson) [reply guide/english/python/class/index.md]: Not only that, but we do accept PRs which only add a url.  We really want new content or improvements to the existing content.  I will close this PR.

**Comment by RandellDawson** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we’ve reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

**Reason:** We do not accept PRs which **only** add links when there are already valid reference links for the article.  Instead of just adding a link, it would be great if you can add original content. Paraphrased content from another source should be referenced.

Thank you and happy coding.

**Pull request closed without merging**